### ISSUE TITLE
Cannot see logs in headless mode

### ISSUE THREAD

**Issue by farzadmf** [WhoIsSethDaniel/mason-tool-installer.nvim#53]
Hi, I have the following command to update the tools in the terminal, using `headless` mode:

```sh
nvim --headless +'autocmd User MasonToolsUpdateCompleted quitall' +'MasonToolsUpdateSync'
```

When I run this command, it runs and exists without no logs, _but_ a coupld of packages have issues to update, which can be seen if I don't run the `autocmd`:

```sh
nvim --headless +'MasonToolsUpdateSync'

pyright: updating to 1.1.370vtsls: updating to 0.2.5
pyright: failed to install
vtsls: failed to install
```

Is there a way I can see the logs in the first example?

Thank you

**Comment by WhoIsSethDaniel** [comment]
By 'logs' you mean printing to `:messages`.  There is an actual log, created by the mason libraries, and typically is located at: `stdpath('log')/mason.log`. You can presumably grep for ERROR messages in the log after running a headless install.

To answer your question: no. Headless mode presumes no output. Perhaps `:h channel-stdio` can be used, but I can't say I'm very familiar with headless mode. I can't think of a simple way around the problem.

**Comment by WhoIsSethDaniel** [comment]
Perhaps one possibility is to pass the status of the update/install to the autocmd along with the names of the packages that were intstalled/updated. Doing so would be a breaking change since the structure of  `data` would change.

**Comment by farzadmf** [comment]
Thank you @WhoIsSethDaniel 

> There is an actual log, created by the mason libraries, and typically is located at: `stdpath('log')/mason.log`

Seems like we were thinking the same on this; I ended up using that file to figure out what's happening.

Thank you for the great plugin; it allows me to keep/maintain all the tools in a single place.

Closing the issue as it's somehow "resolved" by using Mason's log file (at least for my use case) and also the fact that not being able to see the logs is a headless feature/limitation.
